cmd/protecodeExecuteScan.go

```diff
@@ -336,5 +336,12 @@ func getTarName(config *protecodeExecuteScanOptions) string {
 	}
 	// replace unwanted chars
 	fileName = strings.ReplaceAll(fileName, "/", "_")
+	log.Entry().Infof("Filename : %s", fileName)
+	input := "@sha256"
+	if strings.Contains(fileName, input) { 
+		fileName = strings.ReplaceAll(fileName, ":",    "_")
+		fileName = strings.ReplaceAll(fileName, "@sha256", "")
```

Comment:
Why not re-using `input` here. I guess this is semantically justified. By the way: why is that variable named 'input'?

Reply:
var name changed to sha256 instead. We can improve the variable name further if you prefer